integration-cli/docker_cli_run_unix_test.go

```diff
@@ -414,3 +414,19 @@ func (s *DockerSuite) TestTwoContainersInNetHost(c *check.C) {
 	dockerCmd(c, "stop", "first")
 	dockerCmd(c, "stop", "second")
 }
+
+func (s *DockerSuite) TestStopContainerSignal(c *check.C) {
```

Comment:
Why is this test unix-only? Isn't the platform check currently client side only.